lib/parsers/parseOCLC.py

```diff
@@ -218,41 +229,12 @@ def _matchRegexEbook(instance, uri, ebook_regex, uriIdentifier):
             instance.addFormat(**{
                 'source': source,
                 'content_type': 'ebook',
-                'links': [Link(url=uri, mediaType='text/html', flags=bookFlags)],
-                'identifiers': [Identifier(identifier=uriIdentifier)]
-            })
-            return True
-
-
-def _addEbook(instance, holding, subfield, uri, uriIdentifier):
```

Comment:
Why is this not needed anymore?

Reply:
This was a method that was attempting to grab ebooks from anything that identified itself as an ebook. So it was looking in the URI description field for anything that contained "epub" or "ebook" and then added it as an Item regardless of the source. Since we only want these files from specific providers it is much easier to check that the URI matches what we expect since they have consistent formats.

This also has the side effect of excluding links from the providers that we want that don't match the expected format since we can't be sure that we would be able to display content from non-standard links